PlexPy.py

```diff
@@ -1,4 +1,7 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+#!/usr/bin/env python
```

Comment:
It slipped my mind, thanks for the reminder. I'll fix it when it get home from work.